nightwatch.json

```diff
@@ -0,0 +1,17 @@
+{
+    "src_folders": ["tests"],
+    "page_objects_path": "tests/page-objects",
+    "webdriver": {
+        "start_process": true,
+        "server_path": "node_modules/.bin/chromedriver",
```

Comment:
smelling that this relies on a Chrome browser to be available on the host? Need to test it in Github actions

Reply:
We'll raise an issue for adding this to ci with this point in mind.

Reply:
chrome is available, but you need to have the right version of chromedrive (80?)